### PR TITLE
fixed docs for deno

### LINKED ISSUE #936: Deno docs bug
Hi. I found misstake in getting started guide.
It says
```ts
import { Pool } from 'pg-pool'
```
but actually pg-pool exports pool as default export and this code doesnt work
`error: Uncaught SyntaxError: The requested module 'pg-pool' does not provide an export named 'Pool'`
should be
```ts
import Pool from 'pg-pool'
```
https://www.kysely.dev/docs/getting-started?package-manager=deno&dialect=postgresql#instantiation

### PR DESCRIPTION
**Pull request by kolay-v**
fixes #936

### CODE REVIEW

**Comment by kolay-v** [comment]
like so?

File: site/docs/getting-started/Instantiation.tsx [hunk 1]
```diff
@@ -33,10 +33,10 @@ function getDialectSpecificCodeSnippet(
 ): string {
   const driverNPMPackageName = getDriverNPMPackageNames(packageManager)[dialect]
   const dialectClassName = DIALECT_CLASS_NAMES[dialect]
-  const poolClassName = 'Pool'
+  const poolClassName = packageManager === 'deno' ? 'Pool' : '{ Pool }'
```
Comment (igalklebanov) [inline site/docs/getting-started/Instantiation.tsx:36]: This actually "breaks" `pool: new ${poolClassName}({` on L43.

Now it'll print:
```ts
  pool: new { Pool }({
```

File: site/docs/getting-started/Instantiation.tsx [hunk 2]
```diff
@@ -33,10 +33,10 @@ function getDialectSpecificCodeSnippet(
...
 
   if (dialect === 'postgresql') {
-    return `import { ${poolClassName} } from '${driverNPMPackageName}'
+    return `import ${poolClassName} from '${driverNPMPackageName}'
```
Comment (igalklebanov) [inline site/docs/getting-started/Instantiation.tsx:39]:
```suggestion
    return `import ${packageManager === 'deno' ? poolClassName : `{ ${poolClassName} }`} from '${driverNPMPackageName}'
```

Comment (kolay-v) [reply site/docs/getting-started/Instantiation.tsx:39]: double Templar string. maybe create new variable than?

**Comment by kolay-v** [comment]
oh stop

**Review by igalklebanov — APPROVED** [review]
LGTM! 🚀

**Pull request merged**